src/js/UI.js

```diff
@@ -0,0 +1,128 @@
+import { selector } from "./utils/consts.js";
+import Lotto from "./Lotto.js";
+
+class UI {
+  
+  #lottoPurchaseForm = selector('.lotto-purchase-form');  
+  #lottoPurchaseInput = selector('.lotto-purchase-input');
+  #lottoPurchaseBtn = selector('.lotto-purchase-btn');
+  #ticketContainer;
+  #countedTicketLabel;
+  #lottoTicketUI;
+  #switchBtn;
+  #toggle;
+  #randomNumberLists;
+
+  setEvent() {
+    this.#lottoPurchaseForm.addEventListener('submit', (event) => {
+      event.preventDefault()
+    })
+
+    this.#lottoPurchaseInput.addEventListener('keydown', (event) => {
+      if (event.key === 'Enter' && event.target.value !== "") {
```

Comment:
```suggestion
      if (event.key !== "Enter" || event.target.value === "") return;
```
블록이 깊어질수록 가독성이 좋지 않아지기 때문에
이렇게 if 안으로 진입하지 못하도록 해주면 어떨까요?